locallibs/install.py

```diff
@@ -70,10 +83,15 @@ def install_requirements(requirements_file, framework_path, version):
     subprocess.check_call(cmd)
 
 
-def install_extras(framework_path, version="2.7", requirements_file=None):
+def install_extras(framework_path, install_wheel, upgrade_pip, version="2.7",
+                   requirements_file=None):
```

Comment:
install_wheel and upgrade_pip should go at the end and have default values, like:
```python
def install_extras(framework_path, version="2.7",
                    requirements_file=None, install_wheel=False, upgrade_pip=False)
```